flexmeasures/ui/templates/views/sensors.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>FlexMeasures</title>
+    <meta charset="utf-8">
+
+    <script src="//code.jquery.com/jquery-3.6.0.min.js"></script>
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap-theme.min.css">
+    <script src="//netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js">script></script>
+
+    <script src="https://d3js.org/d3.v6.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega@5"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-lite@5.0.0"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-embed@6.17.0"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.7/jstz.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/litepicker/dist/litepicker.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/litepicker/dist/plugins/ranges.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/litepicker/dist/plugins/keyboardnav.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/litepicker/dist/plugins/mobilefriendly.js"></script>
+  </head>
+  <body>
+
+      <div class="charts text-center">
+          <div class="row"><div class="alert alert-info" id="tzwarn" style="display:none;"></div></div>
+          <div class="row"><div id="datepicker"></div></div>
+          <div class="row"><div id="sensorchart"></div></div><hr>
+      </div>
+
+    <!-- Render Charts -->
+    <script type="text/javascript">
+
+      let vegaView;
+
+      async function embedAndLoad(chartSpecsPath, elementId, datasetName) {
+        var opt = {
+          mode: 'vega-lite',
+          renderer: 'svg',
+          actions: {export: true, source: true, editor: true},
+        };
+
+        await vegaEmbed('#'+elementId, chartSpecsPath + '?dataset_name=' + datasetName, opt)
+        .then(function (result) {
+          // result.view is the Vega View, chartSpecsPath is the original Vega-Lite specification
+          vegaView = result.view;
+        });
+      }
+
+      // Date range utils
```

Comment:
Note: these should probably move to our general `flexmeasures.js` in the future.

Reply:
Please check whether I did this correctly in commit https://github.com/SeitaBV/flexmeasures/pull/99/commits/1a238ef172085a3f4c1aeac9bd6cfbe87c86a855. I used an import for which [I had to make the local script into a module](https://stackoverflow.com/questions/48211891/import-functions-from-another-js-file-javascript#48212558).

Reply:
It looks correct to me. At least, it looks like code we had in a former js-heavy project.